src/app/services/nearbyCities.service.js

```diff
@@ -1,6 +1,7 @@
 export default class NearbyCitiesService {
-    constructor($http) {
+    constructor($http, $state) {
         this.$http = $http;
+        this.$state = $state;
```

Comment:
why is $state injected in this service?

Reply:
I thought that I used the state to change to other view, but no. I already deleted that state.